src/main/java/frc/robot/subsystems/ElevatorSubsystem.java

```diff
@@ -75,23 +76,31 @@ public double getHeight() {
         + ElevatorConstants.kElevatorLowestHeight);
   }
 
-  // Possible print method for heightGet:
-  /*
-  System.out.println("Encoder: " + m_elevator1Encoder.getPosition() + " rotations");
-  System.out.println(
-      "Elevator Relative Height: "
-          + (m_elevator1Encoder.getPosition() * ElevatorConstants.kElevatorHeightToRot)
-          + " inches");
-  System.out.println(
-      "Elevator Height: "
-          + ((m_elevator1Encoder.getPosition() * ElevatorConstants.kElevatorHeightToRot)
-              + ElevatorConstants.kElevatorLowestHeight)
-          + " inches"); */
+  // Calculated equation based on demo speeds and heights
+  // Sheet used for calculation in software drive
+  public double safetyheight(double x) {
+    return (79.1
```

Comment:
writing like this might be difficult to tweak especially in a time crunch.  How about switching it to a set of points like in the PR description and do either linear interpolation or spline interpolation between those points?